lib/mix/tasks/daily_checks.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Mix.Tasks.DailyChecks do
+  @moduledoc """
+  Runs daily checks.
+  """
+
+  use Mix.Task
+  alias Samuel.Events
+
+  def run(_args) do
+    Events.respond(tick_event)
+  end
+
+  defp tick_event do
+    %{
+      "action" => "daily"
```

Comment:
I find the fact we're building up what looks a bit like the github webhook here a bit strange.